### DISCUSSION TITLE
Failing to build grammars on macOS Sonoma 14.2.1 (M1)

### DISCUSSION THREAD

**Discussion by bram-dingelstad** [helix-editor/helix#9178] [Troubleshooting]
Hey Helix folks 👋✨

I'm struggling to be able to build grammars because of a messed up dependency on macOS Sonoma 14.2.1, I've tried googling for a while trying most of the usual solutions.

I thought posting it here might help with finding a solution ✨

The log details all of the output from `hx -g fetch` and `hx -g build`

[log.txt](https://github.com/helix-editor/helix/files/13779750/log.txt)

**Comment by pascalkuthe** [comment]
This seems to be so.ekind of weird tool chain issue with the c++ standard library not being available properly. I suspect this is some kind of installation issue on your end since this works on our CI runner.

**Reply by bram-dingelstad** [reply]
yeah, that much I suspected and confirmed before posting, would love to hear if anyone found a way to fix it! :)
